City-of-Pori/pori-kada
# PORIJATE-14 Added feature changes to provide a checkbox for place con…

…tent type

To test:

1. drush fra -y && drush cc all
2. Go to place content type and make sure there is waste search checkbox.
